README.md

```diff
@@ -90,6 +92,59 @@ See below for more details.
         python ${PIAA}/setup.py install
         ```
 
+### Docker
+
+[Docker](https://www.docker.com/what-docker) is an application that lets you run existing
+services, in this case POCS, on a kind of virtual machine. By running via Docker you
+are guranteeing you are using a setup that works, saving you time on setup and 
+other issues that you might run into doing a manual install.
+
+Of course, this also means that you need to set up Docker. Additionally, you will
+need to be able to log into our Google Docker container storage area so you can pull
+down the existing image. The steps below should help you to get going.
+
+#### Install Docker
+
+Depending on what operating system you are using there are different ways of getting
+Docker on your system. The Docker [installation page](https://www.docker.com/community-edition) 
+should have all the answers you need.
+
+#### Install gcloud
+
+`gcloud` is a command line utility that lets you interact with many of the Google
+cloud services. We will primarily use this to authenticate your account but this
+is also used, for example, to upload images your PANOPTES unit takes.
+
+See the gcloud [installation page](https://cloud.google.com/sdk/docs/#install_the_latest_cloud_tools_version_cloudsdk_current_version)
+for easy install instructions.
+
+#### Let Docker use gcloud
+
+Docker needs to be able to use your `gcloud` login to pull the PANOPTES images. There
```

Comment:
Do you think that the "advanced-authentication" is still necessary? The page now recommends just using this command:
```
gcloud auth configure-docker
```
After that (assuming I'd not previously installed docker-credential-gcr, which it doesn't appear that I did according to `gcloud components list`), running this command worked:
```
docker pull gcr.io/panoptes-survey/pocs:latest
```

Reply:
This could have changed, I'll check it out.